spec/models/event_spec.rb

```diff
@@ -0,0 +1,55 @@
+require "rails_helper.rb"
+
+RSpec.describe Event do
+  it { is_expected.to have_db_column(:title) }
+  it { is_expected.to have_db_column(:starts_at) }
+  it { is_expected.to have_db_column(:description) }
+
+  it { is_expected.to have_db_index(:user_id) }
```

Comment:
I would personally group these specs by items checked for example:
```
to have_db_index(. . .)
to have_db_index(. . .)

to belong_to(. . .)
to belong_to(. . .)
```
But if you thinks this is fine too.